src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -213,12 +213,25 @@ class SystemCves extends Component {
     };
 
     showStatusModal = cves => {
+
+        let hasDifferentStatus;
+        if (cves.length > 1) {
+            const selectedCves = Array.from(this.props.cveList.data.filter(function (cve) {
```

Comment:
@Razorlazer do you use the _old_  (`function (cve) {} ` ) way to pass the callback for a specific reason? 
if not, can you please use the arrow function (`cve => ( )` ) to be consistent ?